modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc

```diff
@@ -22,3 +22,53 @@ Red Hat OpenShift GitOps Operator automatically creates a ready-to-use Argo CD i
 .. Select the *openshift-gitops-cluster* instance to display the password.
 .. Copy the password.
 . Use this password and `admin` as the username to log in to the Argo CD UI in the new window.
+
+The default Argo CD instance installed by the OpenShift GitOps Operator as well as accompanying controllers can now run on the infrastructure nodes of the cluster by setting a simple configuration toggle.
```

Comment:
This needs to be a different module.
Maybe titled something like: Running the Argo CD instance at the cluster-level
@reginapizza can you confirm that this module helps the Argo CD instance to run at the cluster level instead of the namespace level?

Reply:
@saumeya @jaideepr97 can one of you confirm that this procedure allows the user to run the default Argo CD instance on a cluster, so that we can title the section appropriately? Currently this is what it says, please let us know if there are any concerns with this title.